content/en/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -15,16 +15,26 @@ This page is an overview of Kubernetes.
 {{% /capture %}}
 
 {{% capture body %}}
-Kubernetes is a portable, extensible open-source platform for managing
-containerized workloads and services, that facilitates both
-declarative configuration and automation. It has a large, rapidly
-growing ecosystem. Kubernetes services, support, and tools are widely available.
+Kubernetes is a cloud infrastructure that provides for the deployment and
+orchestration of containerized applications.    
+It is a portable and extensible open-source platform that is capable of 
+managing both containerized workloads and services. Kubernetes
+provides key features such as a declarative configuration and deployment model,
+built in support for application replicas and autoscaling of applications, and
```

Comment:
nit: you could drop `provides` in:
Kubernetes provides key features and instead just list the key features. Same goes for the clause:
`also provides support for ...`

Reply:
I like having a verb there

Reply:
Will fix the provides support for

Reply:
Hi, @bradtopol. Looking good. I like verbs too 😄. I think I meant to say something like,
`The Kubernetes key features are declarative configuration ...`. There are a number of `provides` and `support` sprinkled throughout. Also, when I look at the rendered page now, there appears to be an extra line/spaces before `In addition ...`